lib/uuid.ex

```diff
@@ -81,17 +80,18 @@ defmodule UUID do
   ## Examples
 
   ```elixir
-  iex> UUID.binary_to_string!(<<135, 13, 248, 232, 49, 7, 68, 135, 131,
-              22, 129, 224, 137, 184, 194, 207>>)
+  iex> UUID.binary_to_string!(<<135, 13, 248, 232, 49, 7, 68, 135,
+  ...>        131, 22, 129, 224, 137, 184, 194, 207>>)
```

Comment:
This is no longer valid Elixir. Why the extra `...>`? Same for a few more lines below.